src/SourceBuild/tarball/content/patches/roslyn-analyzers/0003-Add-offline-validation-to-GenerateDocumentationAndCo.patch

```diff
@@ -0,0 +1,85 @@
+From ea7eaff7d3385901d0d54b347f5762639cf80cd0 Mon Sep 17 00:00:00 2001
```

Comment:
@jmarolf - Is this along the lines of what you had in mind to disable the url validation?

Reply:
This looks correct